syntax = "proto2";
package easy_rec.protos;

enum Similarity {
  COSINE = 0;
  INNER_PRODUCT = 1;
  EUCLID = 2;
}
